rust/src/dcerpc/dcerpc.rs

```diff
@@ -948,28 +931,12 @@ impl DCERPCState {
             self.data_needed_for_dir = direction;
         }
 
-        let buffer = match direction {
```

Comment:
self.bytes_consumed needs to reset to 0

And needs to be out of the state structure to be just a regular variable with a scope limited to this function

Reply:
indeed it seems useless to keep it in the state.. thank you!

Reply:
I was wrong. `bytes_consumed` is needed to know how many bytes the Applayer API has consumed in any subsequent call. Keeping it in the State